torchvision/prototype/models/_api.py

```diff
@@ -78,41 +84,37 @@ def __getattr__(self, name):
         return super().__getattr__(name)
 
 
-def get_weight(fn: Callable, weight_name: str) -> WeightsEnum:
+def get_weight(name: str) -> WeightsEnum:
     """
-    Gets the weight enum of a specific model builder method and weight name combination.
+    Gets the weight enum value by its full name. Example: ``ResNet50_Weights.ImageNet1K_V1``
```

Comment:
```suggestion
    Gets the weight enum value by its full name. Example: ``"ResNet50_Weights.ImageNet1K_V1"``
```

(Although I would remove the backticks since we already have the string delimiters)